packages/sdk/src/Incentives/ERC20PeggedVariableCriteriaIncentive.ts

```diff
@@ -651,27 +652,64 @@ export class ERC20PeggedVariableCriteriaIncentive extends DeployableTarget<
    * Builds the claim data for the ERC20PeggedVariableCriteriaIncentivePayload.
    *
    * @public
-   * @param {bigint} rewardAmount
+   * @param {bigint} signedAmount
    * @returns {Hash} Returns the encoded claim data
    * @description This function returns the encoded claim data for the ERC20PeggedVariableCriteriaIncentivePayload.
    */
-  public buildClaimData(rewardAmount: bigint) {
+  public buildClaimData(signedAmount: bigint) {
     return encodeAbiParameters(
-      [{ type: 'uint256', name: 'rewardAmount' }],
-      [rewardAmount],
+      [{ type: 'uint256', name: 'signedAmount' }],
+      [signedAmount],
     );
   }
 
   /**
-   * Decodes claim data for the ERC20PeggedVariableCriteriaIncentive, returning the encoded claim amount.
+   * Decodes claim data for the ERC20PeggedVariableCriteriaIncentive, returning the claim amount.
    * Useful when deriving amount claimed from logs.
    *
    * @public
    * @param {Hex} claimData
+   * @param {?bigint} [_reward]
+   * @param {?bigint} [_maxReward]
    * @returns {BigInt} Returns the reward amount from a claim data payload
    */
-  public decodeClaimData(data: Hex) {
-    return BigInt(decodeAbiParameters([{ type: 'uint256' }], data)[0]);
+  public async decodeClaimData(
```

Comment:
I think it makes sent to keep the decode _just_ decoding, and put this functionality in a secondary helper, wdyt?

Reply:
Makes sense -- will add secondary helper!